src/contexts/parcelContext.js

```diff
@@ -26,19 +26,25 @@ const ParcelContextProvider = props => {
       return;
     }
     setSearching(true);
-    const response = await httpService.searchParcels(
-      parcelExtendedData.searchParams.statusFilter,
-      parcelExtendedData.searchParams.cityFilter,
-      parcelExtendedData.searchParams.nameFilter );
-    logger.log('[ParcelContextProvider] getAllparcelsfromDB response', response);
-    dispatch(loadParcels(response));
 
-    //need to query all parcels not only the search !
-    const cities = (await httpService.getParcelsCitiesDistinct()).sort();
-    dispatch(updateParcelsCities(cities));
-
-    setSearching(false);
-    setRefreshTime(refreshTime + 1);
+    try {
+      const response = await httpService.searchParcels(
+        parcelExtendedData.searchParams.statusFilter,
+        parcelExtendedData.searchParams.cityFilter,
+        parcelExtendedData.searchParams.nameFilter );
+      logger.log('[ParcelContextProvider] getAllparcelsfromDB response', response);
+      dispatch(loadParcels(response));
+  
+      //need to query all parcels not only the search !
+      const cities = (await httpService.getParcelsCitiesDistinct()).sort();
+      dispatch(updateParcelsCities(cities));
+  
+      setSearching(false);
+      setRefreshTime(refreshTime + 1);
+    } catch(e) {
+      logger.log(e);
+      dispatch(parcelsError(e));
```

Comment:
Question: When this happens - do you want to keep trying to load or stop the timer ?

Reply:
AAMOF I wasn't aware of the refresh timer... I guess it's a product question. 🤷‍♂️